torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -759,6 +760,10 @@ class TORCH_API ProcessGroupNCCL : public Backend {
   // The store is used to broadcast the NCCL unique ID of rank 0.
   c10::intrusive_ptr<Store> store_;
 
+  // Reference to the store so that we can coordinate for dumping
```

Comment:
its not specific to coordinating dumping. its an object anyone can use for anything global.  better to say that it is a store without prefixes so keys are global across PGs.. or something.  (and clarify that store_ is prefixes)